lib/src/generated/platform_constants.dart

```diff
@@ -256,6 +257,15 @@ class TxFormFactorEnum {
   static const String other = 'other';
 }
 
+class TxLogLevelEnum {
```

Comment:
The name ends with 'Enum' but it's a class 🤔 Is there a reason why you can't use enums in this case? If so I would recommend you change the name of this class.

Reply:
This is not an enum. Anything in `platform_constants.dart` are constants used for serialization between Dart and Platform side. This is the platform constants representing the LogLevel enum cases in JSON.

It might be clear enough to just call it `TxLogLevel` though. What do you think?

Reply:
I will write some instructions about how serialization works in Ably Flutter

Reply:
I understand from @ben-xD that he will write those instructions in a subsequent PR (see https://github.com/ably/ably-flutter/issues/263), therefore I am resolving this conversation on @ikbalkaya's behalf.

Also, given this comment was on a `generated/` file, I think this only strengthen's my resolve to encourage us to move to a situation where this repository _just_ contains source files. Generated files should be present ephemerally, only at build time. I've created https://github.com/ably/ably-flutter/issues/264 to track this.